### PR TITLE
Proofreading Collaboration in Studio Pro 9 How-to's

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 1]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
 
 ## 3 Making Changes to an Existing Repository
 
 To make changes to an existing repository, follow these guidelines:
 
-* Create a new branch from where you want to base your work on your fork (this is usually the repository's master branch)
-* To quickly create a branch based on master, use `git checkout -b fix/master/my_contribution master`
-	* Avoiding working directly on the master branch is advised
-* Make commits of logical units
+* Create a new branch from where you want to base your work on your fork (this is usually the repository's **master** branch).
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:28]: Noticing a lot of addition of periods to bulleted lists - this is gradually adding more inconsistency to our docs

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 2]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
+* To quickly create a branch based on **master**, use `git checkout -b fix/master/my_contribution master`.
+	* We advise you to avoid working directly on the **master** branch.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:30]: "Avoid working..." is probably sufficient

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 3]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
 
 To submit changes, follow these guidelines:
 
-* Push your changes to the branch in your fork of the repository
-* Submit a pull request to the appropriate repository in the Mendix organization (for more information, see [About Pull Requests](https://help.github.com/articles/using-pull-requests/))
-* Mention the ID's for any issues that are fixed by the changes (for more information, see [Closing Issues Using Keywords](https://help.github.com/articles/closing-issues-via-commit-messages#closing-issues-with-pull-requests))
-* After feedback has been given, Mendix expects a response within two weeks
-	* After two weeks, we may close the pull request if it isn't showing any activity
+* Push your changes to the branch in your fork of the repository.
+* Submit a pull request to the appropriate repository in the Mendix organization (for more information, see [About Pull Requests](https://help.github.com/articles/using-pull-requests/).
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:41]: Missing closing parentheses here and below

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 4]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
+* Mention the ID's for any issues that are fixed by the changes (for more information, see [Closing Issues Using Keywords](https://help.github.com/articles/closing-issues-via-commit-messages#closing-issues-with-pull-requests).
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:42]: What does "ID's" mean? How should we deal with the plural of an acronym?

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 5]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
+* After feedback has been given, Mendix expects a response within two weeks.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:43]: A good example of why periods are not necessary - a sub-list could become the second sentence if the periods are kept. If no periods are used, either keep the sub-bullet. Can you describe another option if no periods are used?

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md [hunk 6]
```diff
@@ -19,29 +19,29 @@ Before starting with this how-to, make sure you have completed the following pre
...
+	* After two weeks, we may close the pull request if it is not showing any activity.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:44]: Good breaking up of contraction.

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 7]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
 ## 3 Type of Database
 
-The built-in database is a flatfile database called HSQLDB (HyperSQL DataBase). This is the leading SQL relational database software written in Java. It offers a small, fast multithreaded and transactional database engine with in-memory and disk-based tables and supports embedded and server modes. It includes a powerful command line SQL tool and simple GUI query tools.
+The built-in database is a flatfile database called HSQLDB (HyperSQL DataBase). This is the leading SQL relational database software written in Java. It offers a small, fast, multithreaded, and transactional database engine with in-memory and disk-based tables and supports embedded and server modes. It includes a powerful command line SQL tool and simple GUI query tools.
 
-HSQLDB has been constantly developed over 12 years and is used as a database and persistence engine. It is known for its small size, ability to execute completely or partly in memory, its flexibility and speed.
+HSQLDB has been constantly developed over 12 years and is used as a database and persistence engine. It is known for its small size, ability to execute completely or partly in memory, its flexibility, and speed.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:25]: Can probably remove second "its"

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 8]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
 
-Therefore this type of database is excellent to use while developing and running applications in Mendix Studio Pro on a local machine. Thanks to this built in functionality, the developer doesn't have to run a local database engine and management tools.
+Therefore this type of database is excellent to use while developing and running applications in Mendix Studio Pro on a local machine. Thanks to this built-in functionality, the developer does not have to run a local database engine and management tools.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:27]: Missing punctuation after "Therefore"

Comment (Adam-Dupaski) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:27]: Good addition of hyphen to make "built-in" an adjective

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 9]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
-2.  Select the preferred database configuration and click **Make active**.
+1.  Double-click **Settings** in the **App Explorer** to open the **App Settings** dialog box.
+2.  On the **Configurations** tab, select the preferred database configuration in the grid, and click **Make active**.
   
     ![](attachments/18448637/18580427.png)
 
 ## 5 Starting the Database Viewer
 
-To start the built-in database viewer, the following steps have to be applied:
+To start the built-in database viewer, do the following steps:
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:40]: There is a better verb option or construction than "do" for following steps

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md (hunk 5 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md:43]: Move the sub-list one level up and make it a clause of the bullet list, introduced by an em dash?

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 10]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
 
-1.  Run the application locally (if this  is the first time, Studio Pro will ask you to create the new database, so click **Yes**):
+1.  Run the application locally (if this is the first time, Studio Pro will ask you to create the new database, click **Yes** when this occurs):
 
     ![](attachments/18448637/18580426.png)
 
-2.  When the application is running, open the built=in database viewer:
+2.  When the application is running, in the **Console** pane, select **Advanced** > **Start built-in database viewer** to open the built-in database viewer:
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:46]: Cross-reference could be added?

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 11]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
 
     ![](attachments/18448637/18580425.png) 
 
-3.  This will result in the following screen being displayed (depending on the data model of the application):
+    The **Database Manager** dialog box will be displayed (depending on the data model of the application):
 
     ![](attachments/18448637/18580424.png)
 
-This is the database manager. On the left pane all tables from all modules in the running application are showed. On the right top pane it is possible to enter SQL queries and on the right lower pane the results from the entered SQL query will be shown.
+On the left pane, all tables from all modules in the running application are showed. On the upper-right pane, it is possible to enter SQL queries and on the lower-right pane, the results from the entered SQL query will be shown.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:54]: "are showed" is the incorrect case - "showed" needs to be changed

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 12]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
 
 ## 6 Executing the Queries
 
-Queries can be executed in two ways. It can be done directly from the command line (top right pane) in SQL script, or via the explorer on the left pane. For more information about SQL, visit the following page: [http://www.w3schools.com/sql/](http://www.w3schools.com/sql/). To execute queries from the database explorer (left pane), the following steps have to be applied:
+You can execute queries in two ways:
+
+* Directly from the command line on the upper-right pane using SQL script (for more information about SQL, see [W3Schools SQL Tutorial](http://www.w3schools.com/sql/))
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:60]: Is this a valuable third-party link to keep?

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 13]
```diff
@@ -20,56 +20,62 @@ To get the most out of the built-in database, make sure you have completed the f
...
+
+* Via the database explorer on the left pane
+
+To execute queries from the database explorer, do the following steps:
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:64]: "do" can be reconsidered here as well

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 14]
```diff
@@ -11,84 +11,104 @@ Delivering your Mendix app in multiple languages is an important capability for
 
 **This how-to will teach you how to do the following:**
 
-* Add a new language to your app
-* Change the working language
-* Translate individual items of text
-* Translate multiple items of text in one operation
-* Change multiple items of text in one operation
-* Copy a translation from one language to another
+* [Add a new language to your app](#add-new-language)
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:14]: Interesting idea. Do we need this if we have a floating TOC? Are there too many items in the list to begin with?

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 15]
```diff
@@ -11,84 +11,104 @@ Delivering your Mendix app in multiple languages is an important capability for
...
+* [Change the working language](#change-working-language)
+* [Translate individual items of text](#translate-individual-items-of-text)
+* [Translate multiple items of text in one operation](#translate-multiple-items-of-text)
+* [Change multiple items of text in one operation](#change-multiple-items-of-text)
+* [Copy a translation from one language to another](#copy-to-a-new-language)
 
-It is assumed that you know the basics of creating new apps and editing them.
+## 2 Prerequisites
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:21]: Are these good/necessary prerequisites specific to this how-to? Typo with "edite"

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 16]
```diff
@@ -87,15 +93,15 @@ The data is now committed to the team server and can be used by other team membe
 
 To import a data snapshot into the model, the app has to be updated since the last database snapshot has been committed by another team member. To get the data from the committed database snapshot the following steps have to be applied:
 
-1.  First the application has to be updated, so click **Update* on the **Changes** tab:
+1.  Click **Update** on the **Changes** tab to update the application.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/sharing-the-development-database.md:96]: Why a period here and a colon below? Better to be consistent?

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 17]
```diff
@@ -11,84 +11,104 @@ Delivering your Mendix app in multiple languages is an important capability for
...
 
-When you create an app from a Mendix template, there is one language available. This is **English, United States**, which you will usually set to be the default language. This is the language which will be displayed when an end-user has no language selected, or when your app does not support their selected language.
+When you create an app from a Mendix template, there is one language available. This is **English, United States**, which you will usually set to be the **Default language**. This is the language which will be displayed when an end-user has no language selected, or when your app does not support their selected language.
 
 To create your app, do the following:
 
-1. Create a new app using the **Blank App** template. If given the option, leave the default language as **English, United States**.
+1. Create a new app using the **Blank App** template. If given the option, leave the **Default language** as **English, United States**.
 2. Open the app in Studio Pro (images here are from Studio Pro version 8.10).
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:37]: Mx8.10 screenshots. Should these be updated for Mx9? Do the screenshots add enough value to make this a maintenance dependency?

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 18]
```diff
@@ -11,84 +11,104 @@ Delivering your Mendix app in multiple languages is an important capability for
...
 3. Create a domain model in MyFirstModule with two entities:
+
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:39]: A general note - be sure to look at these docs after the PRs are merged to make sure the changes and formatting look okay

Comment (Adam-Dupaski) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:39]: That'll especially be important for the formatting in a section like this, with a sub-list and an image as well. We want to make sure all of this renders fine, so double-checking after a merge is imperative (or building locally to check before merging).

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 19]
```diff
@@ -11,84 +11,104 @@ Delivering your Mendix app in multiple languages is an important capability for
...
 	* **Order** with the integer attribute **OrderNumber**
 	* **Product** with the string attribute **ProductName**
+
 		![Domain model for How To](attachments/translate-your-app-content/domain-model.png)
+
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:44]: Inclusion of "..." in button names is generally unecessary

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md]: Removed periods.

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md]: Indeed. Changed the text.

File: content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md (hunk 3 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/contribute-to-a-github-repository.md]: Added the closing parentheses.

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 7 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:25]: Agreed. I removed it.

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 8 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:27]: Added a comma after "Therefore".

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 9 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:40]: Changed "do" to "follow".

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 13 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:64]: Changed it to "follow".

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 14 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:14]: True, I did not think about the floating TOC. Then the cross-references are not necessary. 

This topic contains multiple how-to's, maybe it is a good idea to split them into separate topics, with "Translation" or "Translate Your App Content" as their parent?

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 16 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:96]: Changed the colon to a comma make it consistent.

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md [hunk 20]
```diff
@@ -87,15 +93,15 @@ The data is now committed to the team server and can be used by other team membe
...
-2.  To implement the data snapshot, the database has to be extracted from a ZIP file to the deployment directory:
+2.  To implement the data snapshot, extract the database from a ZIP file to the **deployment** directory:
 
     ![](attachments/18448637/18580417.png)
 
-3.  Now copy the data directory to the data directory in the deployment directory
+3.  Copy the **data** directory to the **data** directory in the **deployment** directory.
 
 {{% alert type="warning" %}}
 Make sure the names of the databases aren't the same or make a copy of your own database first so it will not be overwritten.
```
[the remark this answers is not in the document]
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md:107]: Fixed.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 21]
```diff
@@ -18,30 +18,43 @@ Delivering your Mendix app in multiple languages is an important capability for
...
 4. Right-click one of the entities and select **Generate overview pages…**.
 5. Ensure both entities are selected and click **OK**.
+
 	![Options selected for Generate Pages](attachments/translate-your-app-content/generate-pages.png)
+	
 	This will generate some pages containing text for you to translate.
+
 6. Open the page **Product_NewEdit** and add a new **Button** widget.
 7. Label the new button **Order**. This button will be used to allow the end-user to order the product they are viewing. Ignore any warnings about the setup of this button.
+
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:54]: Need a line break after the image below

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 22]
```diff
@@ -50,17 +63,22 @@ You now have an app with some translatable texts.
...
+If you want to create the whole app in a language other than the current default, it is best to do this before you start creating the app. If you want to add alternative languages, it is best to do this when your app is nearing completion, as it reduces the amount of work needed to do the translation from the default language, especially if there is already a translation for that language.
 
 To add an additional language, do the following:
 
-1. Select the menu item **Language > Language settings…**.
-2. Click the **Add** button and select a second language. In this example we will be using *Russian, Russia*, but you can use any language you like.
-	{{% alert type="info" %}}It is recommended that you **don't** choose *Dutch, Netherlands* as this already contains translations for many of the translatable texts.{{% /alert %}}
+1. Select the menu item **Language** > **Language settings…**.
+2. Click the **Add** button and select a second language. In this example, we will be using **Russian, Russia**, but you can use any language you like.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:71]: We don't need "click the Add button" - just "click Add"

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 23]
```diff
@@ -98,54 +118,72 @@ Mendix offers you a way to add a translation for all occurrences of a text at on
...
 
 ### 6.2 Dealing With More Complex Translations
 
 Sometimes a single word in the source language does not have a single translation in the destination language.
 
-Follow the steps below to see how to deal with this situation.
+Follow the steps below to see how to deal with this situation:
 
-1. Follow steps **1** through **6** in the previous section to again see a list of texts in your module which can be translated from *English, United States* to *Russian, Russia*.
+1. Follow steps **1** through **6** in the previous section to again see a list of texts in your module which can be translated from **English, United States** to **Russian, Russia**.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:155]: Adding a cross-reference to which "previous section" would help the user

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 24]
```diff
@@ -98,54 +118,72 @@ Mendix offers you a way to add a translation for all occurrences of a text at on
...
 	You will see that the translations for *Order* have now been split between the one with the translation `заказать` and the one with no translation.
+	
 	![All source texts containing the word 'order' showing that ones with different translations are shown separately. ](attachments/translate-your-app-content/batch-translate-order-split.png)
-5. Now you can change the remaining entries for *Order* to `Заказ`, the Russian for *an order*.
+
+5. Change the remaining entries for *Order* to `Заказ`, the Russian for *an order*.
+
 	![All source texts containing the word 'order' showing two different translations for Order. ](attachments/translate-your-app-content/batch-translate-order-2-translations.png)
-6. Click **Translate** to save the translation and **OK** to close the **Batch translate…** dialog.
+6. Click **Translate** to save the translation and **OK** to close the **Batch translate…** dialog box.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:186]: Line break after image

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 25]
```diff
@@ -156,32 +194,37 @@ Once you have made some translations, you may want to look at just the translati
...
 3. Ensure all text starts with a capital letter by typing the new text in the **Replace with** column.
+
 	![All texts which have been translated into Russian.](attachments/translate-your-app-content/batch-replace-russian-capitals.png)
 
 4. Click **Replace** to update the translation dictionary.
 
 ## 8 Copying to a New Language
 
-Having translated your app, you may want to translate it into another, related, language. For example *Dutch, Netherlands* into *Dutch, Belgium*, *English, United States* into *English, United Kingdom*, or *Russian, Russia* into *Belarusian, Belarus*. Many of the translations will be the same so you don't want to type them again.
+Having translated your app, you may want to translate it into another, related, language, for example **Dutch, Netherlands** into **Dutch, Belgium**, **English, United States** into **English, United Kingdom**, or **Russian, Russia** into **Belarusian, Belarus**. Many of the translations will be the same so you do not want to type them again.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:213]: commas after "another" and "related" are probably unnecessary. I'd consider putting parentheses around the examples.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 26]
```diff
@@ -156,32 +194,37 @@ Once you have made some translations, you may want to look at just the translati
...
 
 To copy our Russian translations into a Belarusian dictionary you can do the following:
 
 1. Add *Belarusian, Belarus* as an app language.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:217]: Italics or bold is best for this language choice?

File: content/howto/collaboration-requirements-management/translate-your-app-content.md [hunk 27]
```diff
@@ -202,7 +245,7 @@ For more information, see [Advanced Language Settings](/refguide/language-settin
 
 You have seen how you can create translations easily and even copy your new translation to a new language in the current app.
 
-But what if you have several apps and want to add Russian to all of them? You might be happy to translate your own modules from scratch, but you won't want to translate the system module, administration module, and Atlas UI all over again as they don't change and you already have perfect translations for these.
+But what if you have several apps and want to add Russian to all of them? You might be happy to translate your own modules from scratch, but you will not want to translate the system module, administration module, and Atlas UI all over again as they do not change and you already have perfect translations for these.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/translate-your-app-content.md:248]: "But" could be removed as the start of the paragraph

I'd add a comma after "again"

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md [hunk 28]
```diff
@@ -64,33 +72,36 @@ Follow these steps to remove your user settings and restart Studio Pro.
 
 Follow these steps:
 
-1.  Install [TortoiseSVN](https://tortoisesvn.net/), as suggested in [System Requirements](/refguide/system-requirements).
+1. Install [TortoiseSVN](https://tortoisesvn.net/), as suggested in [System Requirements](/refguide/system-requirements).
+
+   {{% alert type="warning" %}}
+   Mendix Studio Pro uses the Subversion 1.9 working copy. Previous versions of the Mendix Desktop Modeler used a Subversion 1.7 working copy. These working copy versions are *not* compatible.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md:78]: How will this note render with two paragraphs separated by a Markdown line break? What do you add if you want to make the line break render on the site?

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md [hunk 29]
```diff
@@ -134,14 +145,18 @@ Merging the branch to the main line will result in the following information mes
 A conflict is reported on the app *folder*. This usually means that there is a conflict on the `svn:ignore` property.
 {{% /alert %}}
 
-To resolve a conflict on the `svn:ignore` property, perform these steps:
+To resolve a conflict on the `svn:ignore` property, do these steps:
 
 1.  Install [TortoiseSVN](https://tortoisesvn.net/), as suggested in [System Requirements](/refguide/system-requirements).
 
-	{{% alert type="warning" %}}Mendix Studio Pro uses the Subversion 1.9 working copy. Previous versions of the Mendix Desktop Modeler used a Subversion 1.7 working copy. These working copy versions are NOT compatible.	Always use the version of TortoiseSVN that matches your app model. If you open a local model from Mendix version 7.x or 6.x with the latest version of TortoiseSVN, you will no longer be able to open it in Mendix.{{% /alert %}}
+	{{% alert type="warning" %}}
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md:152]: Same question here on the note formatting

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md [hunk 30]
```diff
@@ -134,14 +145,18 @@ Merging the branch to the main line will result in the following information mes
 A conflict is reported on the app *folder*. This usually means that there is a conflict on the `svn:ignore` property.
 {{% /alert %}}
 
-To resolve a conflict on the `svn:ignore` property, perform these steps:
+To resolve a conflict on the `svn:ignore` property, do these steps:
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md:148]: Consider a different verb than "do" for steps throughout

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md [hunk 31]
```diff
@@ -167,9 +183,9 @@ You have resolved the conflict and can commit from Studio Pro.
 If you get this error, try the following options:
 
 * In the [Developer Portal](/developerportal/collaborate/team), check whether the user has access to the app:
-	* If they do not have access, invite them to the app
-	* If they do have access, remove them from the app and add them back – this will re-sync the access rules
-* If the above does not work, make sure the [WebDAV protocol](http://www.webdav.org/) is not blocked within your network – this protocol is [required](/refguide/system-requirements) by Studio Pro for [version control](/refguide/version-control) to work, but it might be blocked by your proxy server or other software like a firewall
+	* If they do not have access, invite them to the app.
```
Comment (Adam-Dupaski) [inline content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md:186]: Evaluate use of periods

Need spaces around the en dash below

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 21 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:54]: Fixed.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 22 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:71]: Fixed.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 23 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:155]: Cross-reference added

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 24 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:186]: Fixed

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 25 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:213]: Do we put examples in parentheses in general?

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 26 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:217]: Changed the style to bold.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 27 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:248]: Fixed.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 19 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:42]: Good to know. I removed "..." from the button names in this topic.

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md (hunk 31 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md:186]: Should this be an [em dash](https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes)?

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 16 above)
Comment (Adam-Dupaski) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md]: Verify that it's a period ;)

File: content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md (hunk 31 above)
Comment (Adam-Dupaski) [reply content/howto/collaboration-requirements-management/troubleshoot-version-control-issues.md]: Already discussed

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 14 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md]: Made a jira task about this.

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 15 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md:21]: Remove the prerequisites.

File: content/howto/collaboration-requirements-management/sharing-the-development-database.md (hunk 16 above)
Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/sharing-the-development-database.md]: Verified. :)

File: content/howto/collaboration-requirements-management/translate-your-app-content.md (hunk 17 above)
Comment (Adam-Dupaski) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md]: I see you updated the screenshots, thanks. There are a lot of screenshots in that doc though, so they take up a lot of space. I think a lot of them can be cropped/edited to save space and keep the reader moving through the doc.

For example:
-6.1 > step 5: edit to reduce the amount of dark grey empty space and move the bottom OK/Cancel buttons up
-6.2 > step 3: edit to reduce the amount of dark grey empty space
-7: same for a lot of the screenshots there - you can also consider cropping to the specific UI section relevant to the step

Comment (Luyao-Zhang-1) [reply content/howto/collaboration-requirements-management/translate-your-app-content.md]: I resized the screenshots in the last commit.

**Pull request merged**